src/irmin-pack/import.ml

```diff
@@ -24,9 +24,20 @@ module Int63 = struct
   include Optint.Int63
 
   let t = Irmin.Type.int63
+
+  module Syntax = struct
+    open struct
+      module Int63 = Optint.Int63
+    end
```

Comment:
Since line 24 does `include Optint.Int63`, I believe this `open` can be removed.

Reply:
Yeah, I realized after the first push that this whole mechanism was not needed. Tricked myself with copy and past of the `Int63.add` ... lines.